net/net.go

```diff
@@ -89,3 +130,34 @@ func ParseCIDRs(cidrs []string) (nets IPNets, err error) {
 	}
 	return nets, nil
 }
+
+// ParseIPCIDRs returns a valid IPSet even in case there are parsing
+// errors of some partial provided input cidrs. So recently added
+// bogus values can be logged and ignored at runtime.
+func ParseIPCIDRs(cidrs []string) (*netipx.IPSet, error) {
```

Comment:
The idea is that if you add a bogus entry you do not screw up the route update.
Thinking about kubernetes as an example: 
someone adds a new bogus cidr range and then updated endpoints won't be updated, or even the full route is dropped.
This scenario would be nicely handled like this.

Reply:
I am not sure I get it - in case there is an invalid value in the list this method would return partial result and an error and predicate creation would fail because error is not nil regardless of partial result.

> or even the full route is dropped.

I think this is better than serving request for the IP that was intended to be blocked.

Reply:
I think we should keep it strict and `return nil, err` on the first encountered error

Reply:
Ok let's do it like that. The underlying library thinks differently, which I thought makes sense in our case, too.
I think if you serve Allow Lists and a bogus entry can crash your allow list then it seems a bit unsafe, but we can also do the validation somewhere else, so in our code everything is "clean". I think that's also great.

Reply:
changed